wordpress-mobile/WordPress-iOS
# Update Aztec to version 1.7.0 and GB latest (1.7.0)

To test:
 - Test copy paste from other apps and internally in GB
 - Test insertion and use of all blocks
 - Check focus issues
 - Check editing the Title of the post and press Enter

Note: still need to update the release notes, waiting for the final version GB-mobile to know what changed.

Update release notes:
